lib/builtin.pm

```diff
@@ -1,4 +1,4 @@
-package builtin;
+package builtin v0.001;
```

Comment:
If using a v-string version it would be written `v0.1` as leading zeroes in this form are insignificant.

Reply:
Fair point. I'll force-push an amend,

Reply:
Turns out: a v-string requires 3 components to it or else it gets upset. I've instead written it as just `0.001`